lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -3,7 +3,7 @@
 Run these tests @ Devstack:
     rake fasttest_lms[common/djangoapps/api_manager/courses/tests.py]
```

Comment:
This line is out of date.  The file location is wrong, and I doubt that there is any Rake configuration at the moment, at least not out of the box.  It should probably be

    paver test_system -fasttest -t lms/djangoapps/api_manager/courses/tests.py

instead, but even better simply removed, so it can't get out of date again.